awx/ui_next/src/components/LaunchButton/LaunchButton.jsx

```diff
@@ -125,16 +133,21 @@ class LaunchButton extends React.Component {
         const { data: job } = await relaunch;
         history.push(`/jobs/${job.id}/output`);
       } else {
-        this.setState({ promptError: true });
+        // TODO: restructure (async?) to send launch command after prompts
+        // TODO: does relaunch need different prompt treatment than launch?
```

Comment:
Yea relaunch is different from a regular launch.  When relaunching a job we don't let the user change any of the previously applied parameters.  As far as I know, the only thing that's promptable on _relaunch_ are credential passwords (if required on launch).